ambari-server/src/main/java/org/apache/ambari/server/scheduler/AbstractLinearExecutionJob.java

```diff
@@ -125,6 +129,19 @@ public void execute(JobExecutionContext context) throws JobExecutionException {
       return;
     }
 
+    String status = null;
+    try {
+      status = executionScheduleManager.getBatchRequestStatus(jobDataMap.getLong(BATCH_REQUEST_EXECUTION_ID_KEY), jobDataMap.getString(BATCH_REQUEST_CLUSTER_NAME_KEY));
+    } catch (AmbariException e) {
+      LOG.warn("Unable to define the status of batch request : ", e);
+    }
+
+    if(ABORTED.name().equals(status) || PAUSED.name().equals(status)) {
+      LOG.debug("The linear job chain was paused or aborted, not triggering the next one");
```

Comment:
Lets make this info, not too many of these will populate the log, right?

Reply:
agree